views/apartments/show.erb

```diff
@@ -1 +1,48 @@
-This is info about apartment #3.
+<h1>
+  Apartment: <%= @apartment %>
+  <a href="/apartments/<%= @apartment.id %>/edit">(edit)</a>
+</h1>
+<section class="apartment">
+  <p>
+    <label>Address</label>
```

Comment:
Generally, you only use `label` to refer to `input` elements -- you don't use it by itself.